lib/health_cards/chunking.rb

```diff
@@ -23,11 +23,47 @@ def generate_qr_chunks(jws)
       jws_chunks.map { |c| convert_jws_to_numeric(c) }
     end
 
+    # Assemble jws from qr code chunks
+    def assemble_jws(qr_chunks)
+      if qr_chunks.length == 1
+        # Strip off shc:/ and convert numeric jws
+        numeric_jws = qr_chunks[0].delete_prefix('shc:/')
+        convert_numeric_jws numeric_jws
+      else
+        ordered_qr_chunks = strip_prefix_and_sort qr_chunks
+        ordered_qr_chunks.map { |c| convert_numeric_jws(c) }.join
+      end
+    end
+
+    def get_payload_from_qr(qr_chunks)
```

Comment:
This may depend on some of the refactor code currently in progress, but should the issuer and verifier methods in chunking.rb be split into their own classes instead of living in the same module?

Reply:
Yea I think this will probably depend on the refactor. I think for now the code is fine to live in the `Chunking` module.

Reply:
I agree.